doc/content/gettingStarted/start.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Configure & Start"
+description: ""
+weight: 2
+draft: false
+--- 
+
+## <a name="running">Running the stack</a>
+
+Download our [Docker Compose configuration](../docker-compose.yml) example. The stack can be run without any configuration, but for the purpose of this guide
```

Comment:
This link is broken

Reply:
Made shortcode to dynamically reference file (So it won't point to github when offline).